guides/hack/02-source-code-fundamentals/01-introduction.md

```diff
@@ -1,11 +1,12 @@
 **Topics covered in this section**
 
-* [comments](comments.md)
-* [constants](constants.md)
-* [keywords](keywords.md)
-* [literals](literals.md)
-* [names, user-defined](names.md)
-* [namespaces](namespaces.md)
-* [scope](scope.md)
-* [script inclusion](script-inclusion.md)
-* [variables](variables.md)
+* [program structure](/hack/source-code-fundamentals/program-structure)
```

Comment:
Does adding this really belong on this PR?

Reply:
Well, the program structure page, which was referenced by the original note I was adding wasn't a part of the actual index. And I figured while I was here I would change the format into a better one -- a format that the test suite can read and cause errors on if a page is referenced that was renamed or deleted, etc. So they are related to my original intention -- adding the fb-only (meta-only) note, but then I got too involved with refactoring the code to the meta branding (also another request).